app/components/Dialog/DialogLndCrashed.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Flex, Box } from 'rebass'
+import { FormattedMessage, injectIntl } from 'react-intl'
+import { withRouter } from 'react-router-dom'
+import Delete from 'components/Icon/Delete'
+import { Dialog, Heading, DialogOverlay, Text } from 'components/UI'
+import { useCloseOnUnmount } from 'hooks'
+import messages from './messages'
+
+const DialogLndCrashed = ({ onCancel, lndCrashReason, history, isOpen }) => {
+  const handleClose = () => {
+    onCancel()
+    history.push('/logout')
+  }
+
+  useCloseOnUnmount(isOpen, handleClose)
+
+  if (!isOpen) {
+    return null
+  }
+
+  const header = (
+    <Flex alignItems="center" flexDirection="column" mb={4}>
+      <Box color="superRed" mb={2}>
+        <Delete height={72} width={72} />
+      </Box>
+      <Heading.h1>
+        <FormattedMessage {...messages.lnd_crashed_dialog_header} />
+      </Heading.h1>
+    </Flex>
+  )
+
+  const { code, signal, error } = lndCrashReason
+  const body = (
+    <>
+      <Flex>
```

Comment:
To me it looks like
 ```
<Flex>
        <Text fontWeight="normal" mr={2}>
          <FormattedMessage {...messages.lnd_crashed_dialog_signal} />
        </Text>
        <Text>
          {signal == null ? <FormattedMessage {...messages.lnd_crashed_dialog_unknown} /> : signal}
        </Text>
      </Flex>
```
can be extracted into sub-component safely. It's repeated 3 times, so i'd say it's totally worth doing. What do you think?

Reply:
Updated